categories/repo.go

```diff
@@ -0,0 +1,25 @@
+package categories
+
+import "budget-helper/database"
+
+type CategoryRepo struct {
+	db *database.Database
+}
+
+func NewCategoryRepo(db *database.Database) *CategoryRepo {
+	return &CategoryRepo{db}
+}
+
+func (c *CategoryRepo) Get(id int) (*database.Category, error) {
+	category := database.Category{}
+	err := c.db.First(&category, id).Error
+
+	return &category, err
+}
+
+func (c *CategoryRepo) GetAllWithOwnerId(id int) ([]*database.Category, error) {
```

Comment:
`GetAllWithUserID`

1. Abbreviations are all caps.
2. We have a `User`, not such thing as `Owner`.